# Exploratory Data Analysis

## Packages & Dependencies 

In [10]:
# Run once to install all required packages
# %pip install -r requirements.txt 

# Load packages
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
from pathlib import Path

# Load data
auction = pd.read_csv(Path("data/auction.csv"))
swarovski = pd.read_csv(Path("data/swarovski.csv"))

## Distributional Analysis of Reserve Price (openbid)
In this dataset, 'openbid' is the seller's opening bid, which functions as the reserve price (the minimum price the seller is willing to accept). No bids below it will win the auction.

It's worth noting though that eBay actually has two distinct concepts that often get conflated:
- Opening bid: the starting price publicly visible to bidders, which is what openbid captures here
- Secret reserve price: a hidden minimum that eBay also allows sellers to set separately, which this dataset doesn't include

For this analysis, 'openbid' will serve as the reserve price proxy.

In [14]:
print(auction['openbid'].describe())

count    10681.000000
mean        52.246256
std        168.453245
min          0.010000
25%          1.000000
50%          4.990000
75%         50.000000
max       5000.000000
Name: openbid, dtype: float64


The distribution of reserve price is heavily right skewed, with a median of $\$4.99$ and a maximum value of $\$5,000$. The mean is $\$52.25$, pulled up by a high-end outliers. The 25th percentile at $\$1.00$ tells us a large share of sellers are essentially setting no meaningful floor at all.

In our regressions we will log reserve price to account for the heavily skewed support of our variable. 